roles/apim-analytics-worker/templates/carbon-home/conf/worker/deployment.yaml.j2

```diff
@@ -0,0 +1,512 @@
+    -
+      name: "server.bootstrap.socket.timeout"
+      value: 60
+    -
+      name: "client.bootstrap.socket.timeout"
+      value: 60
+    -
+      name: "latency.metrics.enabled"
+      value: true
+
+  listenerConfigurations:
+    -
+      id: "default"
+      host: "{{ hostname }}"
+      port: {{ wso2_transport_default_port }}
+    -
+      id: "msf4j-https"
+      host: "{{ hostname }}"
+      port: {{ wso2_transport_msf4j_https_port }}
+      scheme: https
+      keyStoreFile: "${carbon.home}/resources/security/wso2carbon.jks"
+      keyStorePassword: wso2carbon
+      certPass: wso2carbon
+
+  senderConfigurations:
+    -
+      id: "http-sender"
+
+siddhi.stores.query.api:
+  transportProperties:
+    -
+      name: "server.bootstrap.socket.timeout"
+      value: 60
+    -
+      name: "client.bootstrap.socket.timeout"
+      value: 60
+    -
+      name: "latency.metrics.enabled"
+      value: true
+
+  listenerConfigurations:
+    -
+      id: "default"
+      host: "{{ hostname }}"
+      port: {{ siddhi_api_default_port }}
+    -
+      id: "msf4j-https"
+      host: "{{ hostname }}"
+      port: {{ siddhi_api_msf4j_https_port }}
+      scheme: https
+      keyStoreFile: "{{ siddhi_api_keystore_file }}"
+      keyStorePassword: {{ siddhi_api_keystore_password }}
+      certPass: {{ siddhi_api_keystore_cert_password }}
+
+  # Configuration used for the databridge communication
+databridge.config:
+    # No of worker threads to consume events
+    # THIS IS A MANDATORY FIELD
+  workerThreads: 10
+    # Maximum amount of messages that can be queued internally in MB
+    # THIS IS A MANDATORY FIELD
+  maxEventBufferCapacity: 10000000
+    # Queue size; the maximum number of events that can be stored in the queue
+    # THIS IS A MANDATORY FIELD
+  eventBufferSize: 2000
+    # Keystore file path
+    # THIS IS A MANDATORY FIELD
+  keyStoreLocation : {{ databridge_config_keystore_location }}
+    # Keystore password
+    # THIS IS A MANDATORY FIELD
+  keyStorePassword : {{ databridge_config_keystore_password }}
+    # Session Timeout value in mins
+    # THIS IS A MANDATORY FIELD
+  clientTimeoutMin: 30
+    # Data receiver configurations
+    # THIS IS A MANDATORY FIELD
+  dataReceivers:
+  -
+      # Data receiver configuration
+    dataReceiver:
+        # Data receiver type
+        # THIS IS A MANDATORY FIELD
+      type: Thrift
+        # Data receiver properties
+      properties:
+        tcpPort: '{{ thrift_data_receiver_tcp_port }}'
+        sslPort: '{{ thrift_data_receiver_ssl_port }}'
+
+  -
+      # Data receiver configuration
+    dataReceiver:
+        # Data receiver type
+        # THIS IS A MANDATORY FIELD
+      type: Binary
+        # Data receiver properties
+      properties:
+        tcpPort: '{{ binary_data_receiver_tcp_port }}'
+        sslPort: '{{ bianry_data_receiver_ssl_port }}'
+        tcpReceiverThreadPoolSize: '100'
+        sslReceiverThreadPoolSize: '100'
+        hostName: {{ hostname }}
```

Comment:
We can change this to something like `binary_data_receiver_hostname` which makes easier for a user to understand?

Reply:
`binary_data_receiver_hostname` is the hostname of the worker. Therefore I have used worker hostname so that the user does not need to change it in both places.